_posts/2022-06-13-connecting-the-pods.md

```diff
@@ -0,0 +1,291 @@
+This allows the organization to represent itself as a group of smaller organizations. 
+Each of these pods is autonomous and can function on behalf of the business as a whole.
+
+In a traditional company with divisions, the need for a Corporate Strategy occurs when an enterprise develops multiple divisions (typically linked to separate value chains), each with a distinct business strategy. 
+This is for example the case when a company starts diversifying its products or services. 
+Based on how the corporate strategy is handled across these divisions, we can segment the types of diversification into 4 main categories. 
+Single Businesses are centralized and can be defined as having a predominant business strategy that applies to almost all of their business units. 
+Dominant Businesses still have business strategies that vary and take up a significant part of the enterprise, but overall they follow one centralized strategy that accounts for most of their enterprise. 
+Next, we have the diversified businesses, which are comprised of several business strategies, be they related to each other (for example they are all still targeting the technological markets) and thus able to share a value chain, or be they completely unrelated (typically conglomerates). 
+The podular organization attempts to take this conglomerate idea to the next level, and apply it within individual divisions.
+
+{:refdef: style="text-align: center;"}
+<img src="{{ '/img/2022-06-13-connecting-the-pods/strategies02.png' | prepend: site.baseurl }}" alt="Workspace" class="image fit" style="margin:0px auto; max-width:100%">
+{: refdef}
+
+So what is a pod? 
+A small, autonomous unit that has the mandate and the capabilities to deliver whatever it is that its customers value. 
+It is designed to take decisions and implement changes as quickly and as close to the customer as possible. 
+Whereas the traditional company employs processes that can be seen as recipes or a series of steps that need to be executed, the podular organization is structured as a net, distributing the workload based on goals instead of allocating individual steps or stages. 
+And the pods are not necessarily part of the same organization or company but could extend across several to couple its inherent benefits with more economy-of-scale oriented benefits.
+
+This gives a number of advantages based on its lattice-like structure, exchanging consistency and efficiency for flexibility and creativity:
+* Pods are flexible: changing the structure and purpose is easy and fast.
+* Pods are allowed to fail as the network will provide redundancy by adjusting the purpose of similar pods.
+* Pods can scale up fast by adding additional pods and letting them immediately contribute.
```

Comment:
Are we still talking about businesses and not distributed systems? 😄 I love this!

Reply:
:)